content/md/administration/catalog-volume-monitoring_en.md

```diff
@@ -43,7 +43,7 @@ We wrote an [article](https://medium.com/akeneo-labs/because-your-product-catalo
 
 ::: tips
 You can't see the widget we are talking about? What a shame!  
-If you are a Community Edition user or working on the Premise/PAAS version, it's probably because you or your administrator didn't launch the CRON job to calculate the values shown in this widget.  
+If your PIM is on Premise or on our PAAS version, it's probably because you or your administrator didn't launch the CRON job to calculate the values shown in this widget.  
```

Comment:
Why did you remove the Community Edition version? It's also true with the CE version.

Reply:
It's also true with the enterprise edition.
Actually we are mixing two things: CE/EE and on premise/PAAS. It is two distincts things and it's quite confusing the way it's done.

Reply:
It's not true for the EE in Serenity mode. That's why I can't say EE on its own. But maybe we can reformulate.

Reply:
That's why I specify on Premise and PAAS version, neither  SAAS nor EE. This way, you cover all the cases.

Reply:
Ok. I thought "on Premise" was EE only. Remove the uppercase "P" in this case.

Reply:
"on premise" means: it's installed and monitored on your internal servers.
Meaning that all CE are "on premise".

Reply:
Yep, that's what Gaël made me realized. But still, remove the uppercase "P" because it can make us think that we only talk of the on-premise EE which is called "On Premise". Even better, use "on-premise", it's how it's really called in English.